src/parser/demonhunter/havoc/CHANGELOG.js

```diff
@@ -1,13 +1,14 @@
 import React from 'react';
 
-import { Mamtooth, Yajinni, Zerotorescue, Viridis } from 'CONTRIBUTORS';
+import { Mamtooth, Yajinni, Zerotorescue, Viridis, Torothin } from 'CONTRIBUTORS';
```

Comment:
add yourself to the CONTRIBUTORS.js file as well

Reply:
added